pyomo/contrib/solver/tests/solvers/test_ipopt.py

```diff
@@ -48,7 +48,7 @@ def test_ipopt_config(self):
         self.assertIsInstance(config.executable, ExecutableData)
 
         # Test custom initialization
-        solver = SolverFactory('ipopt_v2', executable='/path/to/exe')
+        solver = SolverFactory('ipopt', executable='/path/to/exe')
```

Comment:
This would test the "old" interface, wouldn't it?

Reply:
Nope - this PR follows the documentation and leverages the fact that the new interface is registered with the new SolverFactory as `ipopt`.  Previously, the solver interface was being double-registered with the new factory (as both `ipopt` and `ipopt_v2`).